### DISCUSSION TITLE
How do you stop "other" language being detected?

### DISCUSSION THREAD

**Discussion by Lamparter** [github-linguist/linguist#7113] [Q&A] [answered]
I'm having a strange problem where Linguist keeps detecting the language "Other" in [my repo](https://github.com/Lamparter/DevSanx/) when I would prefer for it to just not list that as a language at all.

![image](https://github.com/user-attachments/assets/948504cb-6a70-41af-93ff-beed56df05d6)

There is probably a really easy way to fix this, any tips would be greatly appreciated 🙂

**Comment by lildude** [comment, marked as answer]
GitHub, independently of Linguist, will group all languages that amount to less than 1% if there are more than one into the "Other" you see.

In your case this will be the JavaScript and TypeScript files:

```console
➜ github-linguist DevSanx/
69.35%  29489      C#
17.52%  7448       CSS
12.86%  5467       HTML
0.21%   91         JavaScript
0.06%   27         TypeScript
➜
```

These files are all in your `Copilot` directory.

So if you really don't want to see "Other" (I'm not sure why you'd want to) you need to do one of two things:

- write more JavaScript or TypeScript to bring the total of one of them by bytes of code over 1%
- use an [override](https://github.com/github-linguist/linguist/blob/main/docs/overrides.md) to tell Linguist to exclude these files or directory.

**Reply by Lamparter** [reply]
Thanks so much! This is really useful and much appreciated ❤️
